ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.12-single-node.yaml

```diff
@@ -0,0 +1,137 @@
+    name: knative-eventing-src
+    namespace: openshift
+    tag: knative-v1.9
+  openshift_knative-serving-src_knative-v1.9:
+    name: knative-serving-src
+    namespace: openshift
+    tag: knative-v1.9
+  openshift_release_golang-1.19:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_eventing-istio-src_knative-v1.9:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-istio-src:knative-v1.9
+    openshift_eventing-kafka-broker-src_knative-v1.9:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-kafka-broker-src:knative-v1.9
+    openshift_knative-eventing-src_knative-v1.9:
+      as:
+      - registry.ci.openshift.org/openshift/knative-eventing-src:knative-v1.9
+    openshift_knative-serving-src_knative-v1.9:
+      as:
+      - registry.ci.openshift.org/openshift/knative-serving-src:knative-v1.9
+  to: serverless-operator-src
+- context_dir: .
+  dockerfile_path: serving/ingress/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: knative-openshift-ingress
+- context_dir: .
+  dockerfile_path: knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: knative-operator
+- context_dir: .
+  dockerfile_path: openshift-knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: openshift-knative-operator
+- context_dir: .
+  dockerfile_path: serving/metadata-webhook/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: metadata-webhook
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-sno-412-continuous
+  cron: 0 0 * * 0
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: operator-e2e
+      commands: make install-mesh test-e2e-with-kafka
+      dependencies:
+      - env: KNATIVE_OPENSHIFT_INGRESS
+        name: knative-openshift-ingress
+      - env: KNATIVE_OPERATOR
+        name: knative-operator
+      - env: OPENSHIFT_KNATIVE_OPERATOR
+        name: openshift-knative-operator
+      from: serverless-operator-src
+      resources:
+        limits:
+          memory: 6Gi
```

Comment:
Is this something that can be lowered?

Reply:
This is the same value as we have in all other configs for serverless-operator. It's the limit, not request. So, I'd probably leave it as we know it works. Finding a lower value would probably need several runs and it could hit back later.